Dockerfile

```diff
@@ -36,6 +37,7 @@ RUN groupadd -r bitcoinz && useradd -r -d /bitcoinz -g bitcoinz bitcoinz
 # Install our run dependencies
 RUN apt-get update \
   && apt-get install -y \
+    curl \
```

Comment:
indeed, it doesn't, I actually mindlessly added the curl because I wasn't sure in which the steps the problem was happening, the build log was so long that I couldn't be able to pinpoint where the curl error happened

Reply:
@bitcoinz-pod merge please!